### PR TITLE
edits for new changes

### PR DESCRIPTION
**Pull request by Dinaamagdy**

### CODE REVIEW

**Comment by Dinaamagdy** [comment]
cloudspace testcases

----------------------------------------------------------------------
Ran 8 tests in 381.938s

OK (SKIP=3)

File: tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py [hunk 1]
```diff
@@ -70,11 +74,17 @@ def test002_create_account_with_correct_params(self):
...
         CU_I = randint(15, 30)
         CU_M = randint(15, 30)
-        self.accounts[self.acc1] = {'openvcloud': self.openvcloud, 'maxMemoryCapacity': CU_M,
+        self.accounts[self.acc1] = {'openvcloud': self.openvcloud,
+                                    'name': self.acc1_name,
+                                    'maxMemoryCapacity': CU_M,
                                     'maxCPUCapacity': CU_C, 'maxVDiskCapacity': CU_D,
-                                    'maxNumPublicIP': CU_I }
+                                    'maxNumPublicIP': CU_I,
+                                    'users': OrderedDict([('name', self.vdcuser),
```
Comment (john-kheir) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py]: why users section is back here?

File: tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py [hunk 2]
```diff
@@ -113,15 +118,16 @@ def test003_create_cloudspace_with_different_limitaions(self, type, factor):
         CU_I = randint(2, 1000) * factor
         CU_M = randint(2, 1000) * factor
         CU_NP = randint(2, 1000) * factor
-        self.cloudspaces[self.cs1] = {'account': self.acc1, 'maxMemoryCapacity': CU_M,
+        import ipdb; ipdb.set_trace()
```
Comment (john-kheir) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py]: please remove the ipdb .. and make a full run for the whole file

File: tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py [hunk 3]
```diff
@@ -218,11 +229,9 @@ def test002_start_stop_vm(self):
         self.wait_for_service_action_status(self.vm1, res[self.vm1]['start'])
 
         self.log(" Check that [VM1] is running.")
-        cloudspaceId = self.get_cloudspace(self.cs1)['id']
-        vm = self.get_vm(cloudspaceId, self.vm1)
-        self.assertEqual(vm["status"], "RUNNING")
+        cloudspaceId = self.get_cloudspace(self.cs1_name)['id']
+        self.assertTrue(self.wait_for_vm_status(cloudspaceId, self.vm1_name))
```
Comment (john-kheir) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: why did u removethe check for the vm status

Comment (john-kheir) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: where is wait_for_vm_status fun?

File: tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py [hunk 4]
```diff
@@ -301,7 +309,7 @@ def test004_clone_vm(self):
         self.assertEqual(vm2_c["vcpus"], vm1["vcpus"])
         self.assertEqual(vm2_c["sizeid"], vm1["sizeid"])
 
-    @unittest.skip("Not tested due to environment problems.")
+    @unittest.skip("https://github.com/0-complexity/openvcloud/issues/1518")
```
Comment (john-kheir) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: please check this ticket again .. seems it is working on other envs

File: tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py [hunk 5]
```diff
@@ -375,7 +384,7 @@ def test006_rollback_of_machine(self):
                              vms=self.vms, temp_actions=temp_actions)       
 
         self.log("Check that [sn3] has been deleted.")
-        snapshots_after_rollback = self.get_snapshots_list(self.cs1, self.vm1)
+        snapshots_after_rollback = self.get_snapshots_list(self.cs1_name, self.vm1_name)
         self.assertNotIn(snapshots_before_rollback[2]["name"], [sn["name"] for sn in snapshots_after_rollback])
         self.assertEqual(len(snapshots_after_rollback), 2)
```
Comment (john-kheir) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: the pull request in the skip is merged .. please run the testcase

File: tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py (hunk 1 above)
Comment (Dinaamagdy) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py]: I think it's old version , I will delete it

File: tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py (hunk 2 above)
Comment (Dinaamagdy) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py]: I did , and I added the result on pull request , I think it's old version , sorry I will check it

File: tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py (hunk 3 above)
Comment (Dinaamagdy) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: on G8_testing  framework

Comment (Dinaamagdy) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: https://github.com/0-complexity/G8_testing/pull/396/files

**Pull request merged**